examples/browser/getassetstate/demo/demo.html

```diff
@@ -1,4 +1,4 @@
-<script src="https://cdn.jsdelivr.net/npm/@cityofzion/neon-js@4.5.2/dist/browser.min.js"></script>
```

Comment:
how about using a non-versioned url like `https://cdn.jsdelivr.net/npm/@cityofzion/neon-js` to always get the latest

Reply:
Will do, thanks for the support.